### ISSUE TITLE
warn/handle no-subsystem turrets

### ISSUE THREAD

**Issue by EatThePath** [Baezon/pof-tools#130]
If you set up a turret from scratch it doesn't automatically get the `$special=subsystem` data and subsequently FSO won't recognize it as a valid turret. Adding this data when defining something as a gun object and warning if it's absent both sound like useful things to me.

**Comment by Baezon** [comment]
Yes, I've been procrastinating on handling subobject properties in general, just because there's so much...
